README.md

```diff
@@ -1,8 +1,110 @@
 # iotagent-manager
 
-IoT Agent Manager proxy
+## Index
+
+* [Overview](#overview)
+* [Configuration] (#configuration)
+* [Subscription API](#subscriptionapi)
+* [Development Documentation](#development)
+
+## <a name="overview"/> Overview
+### Description
+The IoT Agent Manager works as a proxy for scenarios where multiple IoT Agents offer different southbound protocols.
+The IoTA Manager appears as a single administration endpoint for provisioning tasks, redirecting provisioning requests
+to the appropriate IoTAgent based on the declared protocol.
+
+The IoTAgent Manager also offers a cache of all the provided device Configurations, to fasten the retrieval of certain
+information from the Agents.
+
+## <a name="configuration"/> Configuration
+The IoT Agent Manager main configuration point is the `config.js` file at the root of the project. The following section
```

Comment:
It's not in this PR, as there is no executable file yet. It will be introduced when the executable is added.